gallery/src/styles/SelectableColors/SelectableColorCustomExample.scss

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+.gallery-example-candy {
+  // The values of these custom properties depend on which .nx-selectable-color--… class is applied
+  background-color: var(--nx-selectable-color-dark);
+  color: var(--nx-selectable-color-light);
+
+  border-radius: 12px / 8px;
+  display: inline-block;
+  height: 16px;
+  width: 24px;
+  line-height: 14px;
+  margin-right: 16px;
```

Comment:
Should probably use the variable here even if it is just an example

Reply:
> Should probably use the variable here even if it is just an example

f10f2910